app/qml/main.qml

```diff
@@ -11,6 +11,13 @@ ApplicationWindow {
     visibility: __appwindowvisibility
     title: qsTr("Input")
 
+    function isPositionOutOfExtent() {
+        return (positionKit.screenPosition.x < 0) ||
+                (positionKit.screenPosition.y < mainPanel.height) ||
+                (positionKit.screenPosition.x > mapCanvas.width) ||
+                (positionKit.screenPosition.y > mapCanvas.height)
+    }
```

Comment:
I wouldn't remove the "border" handling we had before - it could be passed as an argument to isPositionOutOfExtent()